randovania/games/am2r/json_data/Golden Temple.txt

```diff
@@ -338,7 +338,9 @@ Extra - map_name: rm_a1h08
   * Layers: default
   * Open Passage to Outer Temple Save Station/Dock to Spider Ball Chamber
   > Pickup (Spider Ball)
-      Morph Ball
+      All of the following:
```

Comment:
Morph only was intended here, as you can do this:
![grafik](https://github.com/randovania/randovania/assets/38186597/6f27855d-0c7a-4272-bccd-e4a6b4431afd)

Stand on the pillar, morph, and then just move right->left. makes you always fall into the tunnel

Reply:
Was unsure if this rolling into a morph tunnel from above should be considered movement (beginner)?

Reply:
I personally lean towards "as long as it's very easy to do, and there are no negative consequences from trying it, it's fine as trickless"